apps/teams-test-app/src/components/AppInitialization.tsx

```diff
@@ -56,9 +56,12 @@ const NotifyExpectedFailure = (): React.ReactElement =>
           return;
         }
         const acceptableValues = Object.values(app.ExpectedFailureReason);
-        if (!input.reason && !acceptableValues.includes(input.reason)) {
+        if (input.reason && !acceptableValues.includes(input.reason)) {
           throw new Error(`input must be one of: ${JSON.stringify(acceptableValues)}`);
         }
+        if (!input.reason) {
+          input.reason = app.ExpectedFailureReason.Other;
+        }
```

Comment:
Will it ever execute? Given that we have statements from 54-56

Reply:
omg you're right.....i think i just want vacation to come

Reply:
I am so sorry for being sloppy, thank you for your patience